src/normalized.css

```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them.